src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/validator.html.twig

```diff
@@ -1,5 +1,34 @@
 {% extends '@WebProfiler/Profiler/layout.html.twig' %}
 
+{% block head %}
+    {{ parent() }}
+
+    <style>
+        #collector-content .sf-validator {
+            margin-bottom: 2em;
+        }
+
+        #collector-content .sf-validator .sf-validator-context,
+        #collector-content .sf-validator .trace {
+            border: var(--border);
+            background: var(--base-0);
+            padding: 10px;
+            margin: 0.5em 0;
+            overflow: auto;
+        }
+        #collector-content .sf-validator .trace {
+            font-size: 12px;
+        }
+        #collector-content .sf-validator .trace li {
+            margin-bottom: 0;
+            padding: 0;
+        }
+        #collector-content .sf-validator .trace li.selected {
```

Comment:
Those styles are almost a copy/paste of the serializer and messenger ones (except the panel selector). This makes me think it might deserve a reusable component in the main styles instead of re-creating this rendering for each panel that has such a structure (and third-party bundles might also have a similar panel btw and would benefit from it)

Reply:
OK; but let's do it in a separate PR, please. Thanks.